modules/nw-endpoint-route53.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="nw-endpoint-route53_{context}"]
+= Endpoint configuration for route53
+
+The `route53` endpoint URL must use regions, `us-east-1`, `us-west-1`, or no region because Route 53 is not a regionalized service. The `us-east-1` region is required for the `tagging` endpoint URL to get the hosted zone, `route 53`, of the Elastic Load Balancing (ELB).
```

Comment:
I think we can remove the "because Route 53 is not a regionalized service" language, but I'm not sure yet.